controllers/complianceeventsapi/server.go

```diff
@@ -221,6 +240,12 @@ func (s *ComplianceAPIServer) Start(ctx context.Context, serverContext *Complian
 	mux.HandleFunc("/api/v1/compliance-events/", func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 
+		// To verify each request independently
+		userConfig, err := getUserKubeConfig(s.cfg, w, r)
```

Comment:
Could you move this to right before `getSingleComplianceEvent`? I'd rather avoid doing work if the database is offline or the method is invalid.